### PR TITLE
Feature-onboarding completed

### PR DESCRIPTION
**Pull request by justinl-y**
Closes issue #18

### CODE REVIEW

File: js/components/AppOnboarding/index.js [hunk 1]
```diff
@@ -12,9 +12,8 @@ const AppOnboarding = ({ image, text }) => {
         style={styles.background}
       />
       <View
-        style={styles.yellowseperator}
-      >
-      </View>
+        // style={styles.yellowseperator}
+      />
```
Comment (BlakeTurner) [inline js/components/AppOnboarding/index.js]: Empty tag?

File: js/components/AppOnboarding/style.js [hunk 2]
```diff
@@ -2,17 +2,21 @@ import { StyleSheet } from 'react-native';
...
     alignItems: 'center',
   },
-  yellowseperator: {
-    flex: 0.25,
-    height: 165,
-    backgroundColor: colors.mainBrandYellow
+  yellowSeperator: {
+    // flex: 0.25,
+    // height: 165,
+    // backgroundColor: colors.mainBrandYellow,
```
Comment (BlakeTurner) [inline js/components/AppOnboarding/style.js]: Kill unused styles

File: js/components/SignInForm/index.js [hunk 3]
```diff
@@ -36,9 +36,10 @@ class SignInForm extends Gandalf {
     super(fields);
   }
 
-  componentWillUpdate(nextState) {
-    if (nextState.userSignedIn) {
-      this.props.navigation.navigate('HomeScreenNavigator');
+  componentWillUpdate(nextProps) {
+    if (nextProps.userSignedIn) {
+      // this.props.navigation.navigate('AppOnboardingScreen'); // for testing
```
Comment (BlakeTurner) [inline js/components/SignInForm/index.js]: Kill testing code

File: js/scenes/AppOnboardingScreen/AppOnboardingScreen.js [hunk 4]
```diff
@@ -1,16 +1,88 @@
...
+    this.state = {
+      currentPage: 0,
+    };
+  }
+
+  onScroll = (event) => {
+    const offsetX = event.nativeEvent.contentOffset.x;
+
+    this.setState({
+      currentPage: Math.floor((offsetX - (windowWidth / 2)) / windowWidth) + 1,
```
Comment (BlakeTurner) [inline js/scenes/AppOnboardingScreen/AppOnboardingScreen.js]: This is pretty dense. Will it work if you add more steps? If not, maybe make it into a function that is more dynamic.

File: js/scenes/AppOnboardingScreen/AppOnboardingScreen.js [hunk 5]
```diff
@@ -1,16 +1,88 @@
...
+    });
+  }
+
+  onItemTap = (index) => {
+    console.log(index);
+  }
```
Comment (BlakeTurner) [inline js/scenes/AppOnboardingScreen/AppOnboardingScreen.js]: Usage?

File: js/scenes/AppOnboardingScreen/styles.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import { StyleSheet } from 'react-native';
+// import { windowHeight } from '../../styles/baseStyles';
```
Comment (BlakeTurner) [inline js/scenes/AppOnboardingScreen/styles.js]: Make sure you clean this sort of stuff out before submitting a PR.

**Review by BlakeTurner — CHANGES_REQUESTED** [review]

File: js/components/AppOnboarding/index.js (hunk 1 above)
Comment (justinl-y) [reply js/components/AppOnboarding/index.js]: Done.

File: js/components/AppOnboarding/style.js (hunk 2 above)
Comment (justinl-y) [reply js/components/AppOnboarding/style.js]: Done.

File: js/components/SignInForm/index.js (hunk 3 above)
Comment (justinl-y) [reply js/components/SignInForm/index.js]: Done.

File: js/scenes/AppOnboardingScreen/AppOnboardingScreen.js (hunk 5 above)
Comment (justinl-y) [reply js/scenes/AppOnboardingScreen/AppOnboardingScreen.js]: Removed.

File: js/scenes/AppOnboardingScreen/styles.js (hunk 6 above)
Comment (justinl-y) [reply js/scenes/AppOnboardingScreen/styles.js]: Done.

File: js/scenes/AppOnboardingScreen/AppOnboardingScreen.js (hunk 4 above)
Comment (justinl-y) [reply js/scenes/AppOnboardingScreen/AppOnboardingScreen.js]: Spun out into separate function.

**Comment by justinl-y** [comment]
PR issues actioned.

**Review by BlakeTurner — APPROVED** [review]

**Pull request merged**